sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -51,6 +56,12 @@ def _get_match(proto, filter_fn):
   return query[0]
 
 
+# V1b3 MetricStructuredName keys to accept and copy to the MetricKey labels.
+STEP_LABEL = 'step'
+STRUCTURED_NAME_LABELS = [
+    'execution_step', 'original_name', 'output_user_name']
```

Comment:
You can make this a set to have quicker lookups

Reply:
done